content/get-started/index.md

```diff
@@ -0,0 +1,98 @@
+
+Check our [**issues dedicated page**](/issues/) for known issues, problems and tips. Galaxy uses [**GitHub issues**](https://github.com/galaxyproject/galaxy/issues) for tracking down bugs and feature requests, **please make sure you search the list for duplicates before creating a new issue**.
+
+If you couldn't find a solution, reach out to the [**Galaxy Help Forum**](https://help.galaxyproject.org/) as *the* place to ask your questions related to Galaxy use or  [**create an issue**](https://docs.github.com/es/issues/tracking-your-work-with-issues/creating-an-issue) yourself. Either way, we are more than happy to help you out!
+
+### Mailing lists
+If you haven't used mailing list before they are simply a list of addresses to which the same information is being sent, just another way to communicate and interact with the Community. See them [**here**](/mailing-lists/) and remember [**don't be afraid to ask questions**](/mailing-lists#new-to-mailing-lists/)
+
+### Galaxy Publication Library
+Check what others members have done in our [**Publication library**](https://www.zotero.org/groups/1732893/galaxy) which contains publications that use, reference, implement, and extend Galaxy.
+
+## Step 3:  Your place within the Galaxy Community
+
+<img class="img-fluid float-right" src="/images/undraw-illustrations/connected-people.svg" style="width:200px;height:100px;" alt="community events" />
+
+Still wonder what the Galaxy Community is about?
+You already know it, you are already part of it! But let’s dive in the complexity of the [Galaxy Communities](/community/)!
+
+### Regional Galaxy Communities
+From India to Australia, Japan to France and many more countries [**Regional Communities**](/community/#regional-communities) are a big part of Galaxy. Find ways to contact them, platforms and lots of other useful links.
+
+### Galaxy Communities of Practice Communities
+
+Another way communities are formed is by Community such as Proteomics,Transcriptomics,Ecology and Genome annotation to name a few. Most have set up their own servers and mailing list, look for them [**here**](/community/#communities-of-practice)
+
+### Set up your own Community
+What if you can't find yourself in these lists of regional and/or community of practice? Galaxy welcomes new communities, please see [**here**](/get-started/new-leads).
+
+## Step 4: Take part and contribute to Galaxy
+
+[Here's](https://galaxyproject.org/community/contributing/) our guide on how to start contributing and there are many ways to do so: coding, writing and creating tutorials, answering questions in Gitter or the Help Forum and many more.
+
+### Working Groups
+Interested on a particular area within Galaxy? Connect with a [**Working Group**](/community/wg/). Find the repository, Gitter channel and GDrive for each one.
```

Comment:
I don't understand the sentence:

Find the repository, Gitter channel and GDrive for each one.

- This is more about the Github Project where they have some kind of "roadmap" or at least info on they past, current and future work (and not necessarily about the github repository)
- GDrive: I would put Google Drive; in case someone wonders about this "acronym".